ITMO_FS/filters/univariate/UnivariateFilter.py

```diff
@@ -86,16 +71,14 @@ def get_scores(self, X, y, feature_names=None):
                 The training input samples.
             y : array-like, shape (n_samples, )
                 The target values.
-            feature_names : list of strings, optional
```

Comment:
Why not optional?

Reply:
there was an unnecessary input_check in this function even though we already did that in fit (row 130), by the time we call get_scores feature_names is already initialized